src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,139 @@
+@import '../utils/veriables';
+
+main {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  gap: 46px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  main {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  main {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media (min-width: 1024px) {
+  main {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
+
+.product-card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: $card-width;
+  padding: 16px;
+  display: flex;
+  transition: transform $tramsform-time;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-left: 3px;
+    margin-right: 3px;
+    margin-top: 16px;
+  }
+
+  &__title {
+    display: flex;
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 40px;
+    margin-bottom: 0;
+    line-height: 18px;
+    color: $text-color-1;
+    transition: transform $tramsform-time;
```

Comment:
There is a typo in the variable name `$tramsform-time`. It should likely be `$transform-time`. Ensure the variable is correctly defined in the imported file.